tests/functional/lang/eval-fail-addDrvOutputDependencies-multi-elem-context.err.exp

```diff
@@ -0,0 +1,17 @@
+error:
+       … while calling the 'addDrvOutputDependencies' builtin
+
+         at /pwd/lang/eval-fail-addDrvOutputDependencies-multi-elem-context.nix:18:4:
+
+           17|
+           18| in builtins.addDrvOutputDependencies combo-path
+             |    ^
+           19|
+
+       … while calling the 'derivationStrict' builtin
+
+         at /builtin/derivation.nix:9:12:
+       … while evaluating derivation 'fail'
+         whose name attribute is located at /pwd/lang/eval-fail-addDrvOutputDependencies-multi-elem-context.nix:3:5
+
+       error: operation 'addTextToStore' is not supported by store 'dummy'
```

Comment:
Fails for the wrong reason, not really sure what I should do about that?

Reply:
N.B. I think we have a few of these already.

Reply:
If the test requires write access to the store, then it should probably not be here. Maybe add another similar test suite for stuff that requires write access?

Reply:
Fixed via #9168